test/testutil.h

```diff
@@ -648,5 +648,5 @@ X509 *load_cert_pem(const char *file, OSSL_LIB_CTX *libctx);
 X509 *load_cert_der(const unsigned char *bytes, int len);
 STACK_OF(X509) *load_certs_pem(const char *file);
 X509_REQ *load_csr_der(const char *file, OSSL_LIB_CTX *libctx);
-
-#endif                          /* OSSL_TESTUTIL_H */
+time_t test_asn1_string_to_time_t(const char *asn1_string);
+#endif                          /* OSSL_TESTUTIL_H */
```

Comment:
Thank you for removing the comment block (such commentary about changes in the code base happens naturally by looking at the git commit history).

However, this last change has lost the trailing newline (\n) in the file - can you put it back, please? The CI is complaining:

```
In file included from test/testutil/apps_shims.c:12:
test/testutil/../testutil.h:652:54: error: no newline at end of file [-Werror,-Wnewline-eof]
#endif                          /* OSSL_TESTUTIL_H */
                                                     ^
1 error generated.
```

Reply:
Updated.

Reply:
Err, have you pushed the change? It's not visible on GitHub here